### PR TITLE
New Feature

### PR DESCRIPTION
**Pull request by iAkashPattnaik**
New Feature As `Stroke`

### CODE REVIEW

File: template/card.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ async function main(name, pic, repo, stars, commits, forks, tis, cis, text_color
             }
         }
     </style>
-    <rect x="0.5" y="0.5" rx="4.5" height="99%" width="494" fill="#${card_color}" stroke="none" stroke-opacity="1" />
+    <rect x="0.5" y="0.5" rx="4.5" height="99%" width="494" fill="#${card_color}" stroke="{stroke}" stroke-opacity="1" />
```
Comment (tuhinpal) [inline template/card.js:63]: Did you read the previous comment ? it not gonna work you need to do this 

${stroke}

**Comment by tuhinpal** [comment]
Added that feature. You can find the changelog here :
https://github.com/cachecleanerjeet/readme-stats-github/commit/b2f72e5fbf977640669fd0ae92fc423b1176cd33

File: template/card.js (hunk 1 above)
Comment (iAkashPattnaik) [reply template/card.js:63]: again the same mistake
i acctually opened the `api/index.js` file and changed there and forgot here

Comment (tuhinpal) [reply template/card.js:63]: I think you don't test that after change

Comment (iAkashPattnaik) [reply template/card.js:63]: na
cause its just 10 lines changed
no need to see small changes

moreover my github homepage was locking bad
so was in a hurry to make this correct

**Pull request closed without merging**